src/OrcasEngine/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -1516,10 +1516,7 @@ internal void AddEventFinished(string projectTargetNames, BuildEventContext buil
                     entryPoint.AddEventFinished(null, buildEventContext, eventTimeStamp);
                 }
 
-                if (startedEvent == null)
-                {
-                    Debug.Assert(startedEvent != null, "Cannot have finished counter without started counter. ");
-                }
+                Debug.Assert(startedEvent != null, "Cannot have finished counter without started counter. ");
```

Comment:
This is in deprecated code. Can you please remove this diff?
Also, `startedEvent` is a field which may be null when this method is called, right?